corehq/apps/accounting/models.py

```diff
@@ -787,7 +787,9 @@ class Meta(object):
     @classmethod
     def get_default_plan_version(cls, edition=None, is_trial=False,
```

Comment:
this might need to be cached now since it will get hit on every request instead of the already cached `get_active_subscription_by_domain`

Reply:
Good point - and it'll be a very small cache too, since there's little variance on these parameters.